store/src/java/com/zimbra/cs/service/mail/CreateCalendarItemException.java

```diff
@@ -107,8 +108,8 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
 
         MailSendQueue sendQueue = new MailSendQueue();
         Element response = getResponseElement(zsc);
-        mbox.lock.lock();
-        try {
+        try (final MailboxLock l = mbox.lock(true)) {
```

Comment:
This causes `sendQueue.send();` to be executed within the lock. You need to make sure that it's executed outside of the lock which means you can't use `try-with-resources` here.